Tools/cases_generator/analyzer.py

```diff
@@ -507,11 +534,15 @@ def compute_properties(op: parser.InstDef) -> Properties:
             tkn.column,
             op.name,
         )
-    infallible = is_infallible(op)
+    pop_error = has_pop_error(op)
+    no_pop_error = has_no_pop_error(op)
+    infallible = not pop_error and not no_pop_error
     passthrough = stack_effect_only_peeks(op) and infallible
     return Properties(
         escapes=makes_escaping_api_call(op),
-        infallible=infallible,
+        pop_error=pop_error,
+        no_pop_error=no_pop_error,
+        # FIX ME!!!
```

Comment:
What's there to fix?

Reply:
The line below, `EXIT_IF` does not imply `DEOPT_IF` https://github.com/python/cpython/pull/116813/commits/9d0d03cfcd3f34c1fd9ce642588861be1e947801